controllers/apply-next/simple/fields.test.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env jest */
+'use strict';
+
+const { filterOptionsBy } = require('../helpers');
+const { legalContactRoles } = require('./fields');
+
+describe('legalContactRoles', () => {
```

Comment:
This test is a bit speculative. It might be better off to have a high level test of enhance form which asserts all rules based on mock submissions, rather than lower level tests like this.